src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -86,7 +86,12 @@ public Task InitLinter(IEnumerable<Rule> rules, CancellationToken cancellationTo
                 Environments = analysisConfiguration.GetEnvironments()
             };
 
-            return MakeCall("init-linter", initLinterRequest, cancellationToken);
+            var responseString = await MakeCall("init-linter", initLinterRequest, cancellationToken);
```

Comment:
Small error handling. The call doesn't throw if there is a problem, it just returns a string with the problem...